### PR TITLE
test: Add selectors part 2

### PR DESCRIPTION
**Pull request by nalmabrcom**
Signed-off-by: Maryna Nalbandian <maryna.nalbandian@broadcom.com>

### CODE REVIEW

**Review by grianbrcom — APPROVED** [review]
:chess_pawn:

File: tests/src/e4e.ts [hunk 1]
```diff
@@ -98,76 +110,86 @@ Cypress.Commands.add('E4EAddNewConnection', (profileName, host) => {
...
-     * Unock the report
-     *
-     * @example cy.deleteE4EConnection(TEST.REPORT.EXAMPLE);
-     */
-    deleteE4EConnection(profileName: string): Chainable<any>;
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      /**
+       * Unock the report
```
Comment (temanbrcom) [inline tests/src/e4e.ts:149]: "Unlock"?

**Review by temanbrcom — APPROVED** [review]
🦠

File: tests/src/e4e.ts (hunk 1 above)
Comment (nalmabrcom) [reply tests/src/e4e.ts]: This helper's comment is not valid, should be `Delete E4E connection`.

**Pull request merged**